packages/integration-sdk-cli/tsconfig.dist.json

```diff
@@ -4,9 +4,7 @@
     "sourceMap": true
   },
   "include": [
-    "src",
-    "src/generator/template/**/*",
-    "src/generator/stepTemplate/**/*"
```

Comment:
Does this work when installing and testing outside of the package. I did this to get those bundled but it's been a while